packages/core/EventManager.ts

```diff
@@ -311,7 +311,31 @@ export default class EventManager {
 
     let createdEvents: EventResult<NewCalendarEventType>[] = [];
     if (event.destinationCalendar) {
-      if (event.destinationCalendar.credentialId) {
+      if (event.schedulingType === "COLLECTIVE" && event.team?.destinationCalendar) {
+        const destinationCalendars = event.team.destinationCalendar;
+        for (const destinationCalendar of destinationCalendars) {
+          if (destinationCalendar.credentialId) {
+            const dbCredential = await prisma.credential.findFirst({
+              where: {
+                id: destinationCalendar.credentialId,
+              },
+            });
+            if (!dbCredential) {
+              console.error("Credential not found for ID:", destinationCalendar.credentialId);
+              continue;
+            }
+            const credential: CredentialWithAppName = {
+              ...dbCredential,
+              appName: "", // Can appName be more easy to get?
```

Comment:
Do we have a right way to obtain appName? I think that value should be on DB? @zomars

Reply:
We could save it in the DB. Let's open a ticket for this.